docs/design/import.md

```diff
@@ -0,0 +1,314 @@
+# Import
+
+Practitioners may have previously existing resources that they wish to bring under Terraform's management. Conceptually, this is called importing as Terraform must create the state of the resource. Providers are responsible for receiving a request for resource import and respond with the relevant resource state(s).
+
+This design documentation will walk through and recommend options for import handling in the framework.
+
+## Background
+
+### Terraform CLI
+
+Terraform CLI supports two forms of interacting with unmanaged resources:
```

Comment:
nitpick

```suggestion
Terraform CLI supports two forms of interacting with resources not in its statefile:
```

"unmanaged" here may be confusing. Does it mean data sources? etc.